wails-frontend/src/layouts/DefaultLayout/DefaultLayout.tsx

```diff
@@ -0,0 +1,9 @@
+import { ReactNode } from 'react';
+
+interface LayoutProps {
+  children: ReactNode;
+}
+
+export function DefaultLayout(props: LayoutProps) {
+  return <div className={`bg-primary min-h-screen`}>{props.children}</div>;
```

Comment:
I would suggest this for the default layout... that way you can just add your children in without any worry of size... and we can keep this as a standard through out the wails screens

<div className="bg-primary flex flex-col justify-center items-center h-screen w-full">
      <div className="w-1/4  max-w-xs  min-w-fit">
      </div>
</div>

Reply:
'' <div className="bg-primary flex flex-col justify-center items-center h-screen w-full">
      <div className="w-1/4  max-w-xs  min-w-fit">
      </div>
</div> '''